packages/dds/tree/src/index.ts

```diff
@@ -170,6 +175,9 @@ export {
     OptionalFieldEditBuilder,
     SequenceFieldEditBuilder,
     SequenceField,
+    singleTextCursor,
+    FieldKinds,
```

Comment:
Exporting FieldKinds like this is problematic since it provides is no way to be selective about what in that module gets exported here.

I don't have a good alternative ( I tried a few things once), but we should revisit this before merging.